swift/Sources/PiranhaKit/CleanupStaleFlags/StaleFlagCleaner.swift

```diff
@@ -327,11 +249,14 @@ class XPFlagCleaner: SyntaxRewriter {
             }
         } else if let functionCallExprNode = FunctionCallExprSyntax.init(node) {
             // handles flag API calls
-            let api = flagApiType(of: functionCallExprNode)
+            guard let api = flagApiType(of: functionCallExprNode) else {
```

Comment:
using a if here removes the need for additional caching calling and also is easier to read. but, fine with me as long as this doesn't affect the functionality.

Reply:
I agree, updated.